### DISCUSSION TITLE
About css import.

### DISCUSSION THREAD

**Discussion by spcma** [valcosmos/val-design#4] [Q&A] [answered]
I found that `index.css` is not compressed, what should I do if I want to import compressed css.

**Comment by valcosmos** [comment, marked as answer]
You can use this directives to compile:

`sass --embed-sources --style compressed ./src/styles/index.scss ./dist/index.min.css`

But I have added `min.css` in the latest version, you can import it directly.

`import 'val-design/dist/index.min.css'`
